test/browser.js

```diff
@@ -3,7 +3,7 @@
 
 // require('./exchange-files')
 // require('./pubsub')
-require('./kad-dht')
+// require('./kad-dht')
```

Comment:
I have disabled them at the time because `daemon.api.id` was returning an empty array for addresses. This way, with no bootstrap peers we would take a long time in order to have the random walk discovery to find the remaining peers.

Reply:
Quick question: Are these tests using the Main Network to test (e.g. use public boostrappers)?

Reply:
No, in this case they are being connected only to each other